mewarpx/tests/test_phist_diagnostic.py

```diff
@@ -0,0 +1,78 @@
+"""Test the mewarpx wrapper for embedded boundaries. This test is the same
+as the test in warpx/Examples/Tests/ElectrostaticSphereEB/inputs_3d but in 2d.
```

Comment:
This is outdated from copying a different test file. Will change along with other suggested changes from PR review.